test/fastify.test.js

```diff
@@ -24,11 +24,10 @@ tap.beforeEach(() => {
 })
 
 test('should log server started messages', async (t) => {
-  await server.listen({ port: 63995 })
+  await server.listen({ port: 63995, host: '127.0.0.1' })
```

Comment:
I changed this, because the previous code was not os-independent.
e.g. ipv6 may not available.